src/recipe.cpp

```diff
@@ -6,6 +6,7 @@
 #include "item.h"
 #include "string_formatter.h"
 #include "output.h"
+#include "overmap.h"
```

Comment:
You don't need to include "overmap.h" here, a forward declarations of `oter_t` and `oter_str_id` is enough to use the functions of the `string_id`.